src/app.js

```diff
@@ -1 +1,23 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const args = process.argv.slice(2);
+
+if (args[0] === 'cp' && args[1] !== args[2]) {
```

Comment:
move it to the `copyFile` & tell the user about error

Reply:
make it with `terminal.question`

Reply:
Don't understand the 'terminal.question'. Googling tells me only about 'readline'.